README.md

```diff
@@ -11,7 +11,7 @@ For more information about the evolving FSH syntax see the [FHIR Shorthand Refer
 
 # Installation for SUSHI Users
 
-SUSHI requires [Node.js](https://nodejs.org/) to be installed on the user's system.  Users should install Node.js 12 (LTS), although the previous LTS version (Node.js 10) is also expected to work.
+SUSHI requires [Node.js](https://nodejs.org/) to be installed on the user's system. Users should install Node.js 12 (LTS), although the previous LTS version (Node.js 10) is also expected to work.
```

Comment:
Glad to have another ally in the war against double space.

Reply:
Oh I see the ally may have been an automatic formatter, but glad to have the ally all the same. And thanks for formatting this, looks more readable now.